### ISSUE TITLE
Vampire Hunters Attack Everything - Epic siege

### ISSUE THREAD

**Issue by PolarisHD** [TeamLapen/Vampirism#374]
- Minecraft:                    1.12.2
- Forge:                       
- Vampirism:                         1.4.9                    

Vampire hunter NPC's attack every entity they find, including villagers

Find a hunter, and wait

It's potentially a compatibility issue, I am running a significant number of mods. As of reporting the issue I will be testing each one to see if I can find if there is a mod that is causing them to behave this way.

**Comment by maxanier** [comment]
This has to be some kind of compatibility issue. I cannot reproduce this with just Vampirism.

Hunter NPCs should only attack creatures that somehow attacked them first, or the entity is somehow a vampire class, which should not be easily possible for normal creatures.

**Comment by Aer-ink** [comment]
are you using dynamic stealth? I got some issues with vampire hunters some versions ago if so try updating it

**Comment by PolarisHD** [comment]
I believe I just solved the issue. 

Anyone running Epic siege mod in conjunction with vampirism needs to have all hunter entities from vampirism set to be exempt from AI modifications in the epic siege mod config. If not, hunters will always be hostile to every entity. 

Sorry if this was a bit of a wasted post, I figured it would be worth saying something if the solution turned out to be more complex than what it was, and/or I couldn't solve it myself.
